src/video/windows/SDL_windowsevents.c

```diff
@@ -356,7 +356,7 @@ static void WIN_UpdateMouseCapture(void)
     }
 }
 
-static void WIN_UpdateFocus(SDL_Window *window, bool expect_focus)
+static void WIN_UpdateFocus(SDL_Window *window, bool expect_focus, DWORD pos)
```

Comment:
Can you split this change into a separate PR, or squash the other commits in this PR so it can be rebased and merged separate commit?